src/net/sourceforge/zmanim/ComplexZmanimCalendar.java

```diff
@@ -89,18 +89,39 @@
  */
 public class ComplexZmanimCalendar extends ZmanimCalendar {
 
+    public static final int SHAAH_ZMANIS_120MINUTES = 2;
```

Comment:
We could extract all the constants into an enum class that is not part of either class and just has all the necessary values.

One advantage is that if another value is inserted in the ZmanimCalendar class, we don't have to worry about remembering to bump the values in this class too. We can also take advantage of the typing and not have to worry about someone using a "random" int value.